Records with bogus epochs should be discarded, not generate alerts (#177)
tlswg/dtls13-spec: draft-ietf-tls-dtls13.md

Using these reserved epoch values a receiver knows what cipher state
has been used to encrypt and integrity protect a

> message. Implementations that receive a records with an epoch value
> for which no corresponding cipher state can be determined SHOULD
> handle it in the same fashion as it handles records which
> cannot be deprotected.

Reviewer: ```suggestion
message. Implementations that receive a record with an epoch value
for which no corresponding cipher state can be determined SHOULD
handle it as a record which fails deprotection.
```
